### PR TITLE
TP-205: Add XML export for Goods Nomenclature.

### PR DESCRIPTION
**Pull request by NDevox**
A JSON API for the Goods Nomenclatures has also
been added for convenience.

### CODE REVIEW

File: commodities/jinja2/taric/footnote_association_goods_nomenclature.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{% from "macros/record.xml" import standard_record %}
+{% from "macros/record.xml" import base_record %}
+
+{% call standard_record(message_counter(), sequence(), transaction_id, record) %}
+    <oub:additional.code.type>
```
Comment (simonwo) [inline commodities/jinja2/taric/footnote_association_goods_nomenclature.xml:5]: This looks copy-pasted? 😜 
Shouldn't this be `footnote.association.goods.nomenclature`? And the fields are different.

File: commodities/tests/test_xml.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import pytest
+
+from commodities.tests.conftest import generate_good
+from common.tests import factories
+from common.tests.util import validate_taric_xml
+
+pytestmark = pytest.mark.django_db
+
```
Comment (simonwo) [inline commodities/tests/test_xml.py:8]: Probably need one for the footnote associate as well then?

File: commodities/jinja2/taric/footnote_association_goods_nomenclature.xml (hunk 1 above)
Comment (NDevox) [reply commodities/jinja2/taric/footnote_association_goods_nomenclature.xml:5]: I wonder how that happened 😏 

I may have forgotten about this one after some "minor" distractions around indents...

File: commodities/tests/test_xml.py (hunk 2 above)
Comment (NDevox) [reply commodities/tests/test_xml.py:8]: Yah that makes sense

File: commodities/jinja2/taric/footnote_association_goods_nomenclature.xml (hunk 1 above)
Comment (NDevox) [reply commodities/jinja2/taric/footnote_association_goods_nomenclature.xml]: Done

File: commodities/tests/test_xml.py (hunk 2 above)
Comment (NDevox) [reply commodities/tests/test_xml.py:8]: Done

**Pull request merged**